doc/users/prev_whats_new/whats_new_0.99.rst

```diff
@@ -28,7 +28,7 @@ toolkit mpl_toolkits.mplot3d (which now comes standard with all mpl
 installs).  See :ref:`mplot3d-examples-index` and
 :doc:`/tutorials/toolkits/mplot3d`.
 
-.. plot::
+.. code-block:: python
```

Comment:
Why did this stop rendering?

Reply:
Because of the `auto_add_to_figure=False`. I figured it was better to keep the code block the same so that is specific to the version of the what's new notes it was written for.